### PR TITLE
Update fields for Treatment type in Clinical History browser

### PR DESCRIPTION
**Pull request by jallentxbiomed**
Changed the category type to "Medication Ordered" so that it would properly overwrite the therapy panel being added in EHR

### CODE REVIEW

File: snprc_ehr/src/org/labkey/snprc_ehr/history/DefaultTreatmentOrdersDataSource.java [hunk 1]
```diff
@@ -35,20 +35,18 @@ protected String getHtml(Container c, Results rs, boolean redacted) throws SQLEx
...
-        addFloatWithUnitsColField(rs, sb, "dosage", "Dosage", "dosage_units");
-        addFloatWithUnitsColField(rs, sb, "volume", "Volume", "vol_units");
-        addFloatWithUnitsColField(rs, sb, "amount", "Amount", "amount_units");
-        addStringField(rs, sb, "performedby", "Ordered By");
-        addStringField(rs, sb, "remark", "Remark");
-        addStringField(rs, sb, "description", "Description");
+        sb.append(safeAppend(rs, null, "code"));
+        sb.append(safeAppend(rs, "Category", "category"));
+        sb.append(safeAppend(rs, "Start Date", "startdate"));
+        sb.append(safeAppend(rs, "Dosage", "dosage"));
```
Comment (spamhurts) [inline snprc_ehr/src/org/labkey/snprc_ehr/history/DefaultTreatmentOrdersDataSource.java]: dosage is in the amount column

File: snprc_ehr/src/org/labkey/snprc_ehr/history/DefaultTreatmentOrdersDataSource.java [hunk 2]
```diff
@@ -35,20 +35,18 @@ protected String getHtml(Container c, Results rs, boolean redacted) throws SQLEx
...
+        sb.append(safeAppend(rs, "Units", "dosage_units"));
```
Comment (spamhurts) [inline snprc_ehr/src/org/labkey/snprc_ehr/history/DefaultTreatmentOrdersDataSource.java]: units is in the amount_units column

File: snprc_ehr/src/org/labkey/snprc_ehr/history/DefaultTreatmentOrdersDataSource.java [hunk 3]
```diff
@@ -26,7 +26,7 @@ public class DefaultTreatmentOrdersDataSource extends AbstractDataSource
 {
     public DefaultTreatmentOrdersDataSource(Module module)
     {
-        super("study", "Treatment Orders", "Therapy", "Therapy", module);
+        super("study", "Treatment Orders", "Medication Ordered", "Therapy", module);
         setShowTime(true);
```
Comment (spamhurts) [inline snprc_ehr/src/org/labkey/snprc_ehr/history/DefaultTreatmentOrdersDataSource.java:34]: We want to stop using table titles - need to change "Treatment Orders" to "treatment_order"

File: snprc_ehr/src/org/labkey/snprc_ehr/history/DefaultTreatmentOrdersDataSource.java [hunk 4]
```diff
@@ -35,20 +35,18 @@ protected String getHtml(Container c, Results rs, boolean redacted) throws SQLEx
...
+        sb.append(safeAppend(rs, "Start Date", "startdate"));
+        sb.append(safeAppend(rs, "Dosage", "dosage"));
+        sb.append(safeAppend(rs, "Units", "dosage_units"));
+        sb.append(safeAppend(rs, "Route", "route"));
+        sb.append(safeAppend(rs, "Frequency", "frequency/meaning"));
+        sb.append(safeAppend(rs, "Duration", "duration"));
+        sb.append(safeAppend(rs, "Reason", "reason"));
+        sb.append(safeAppend(rs, "Remark", "remark"));
+        sb.append(safeAppend(rs, "Description", "description"));
+        sb.append(safeAppend(rs, "End Date", "enddate"));
```
Comment (spamhurts) [inline snprc_ehr/src/org/labkey/snprc_ehr/history/DefaultTreatmentOrdersDataSource.java]: Can you easily format the enddate?

**Review by spamhurts — COMMENTED** [review]
I added a few suggestions

File: snprc_ehr/src/org/labkey/snprc_ehr/history/DefaultTreatmentOrdersDataSource.java (hunk 4 above)
Comment (jallentxbiomed) [reply snprc_ehr/src/org/labkey/snprc_ehr/history/DefaultTreatmentOrdersDataSource.java]: Shouldn't be difficult. Just want the date?

Comment (spamhurts) [reply snprc_ehr/src/org/labkey/snprc_ehr/history/DefaultTreatmentOrdersDataSource.java]: How about the date with the time in hours and minutes?

**Review by spamhurts — APPROVED** [review]
Looks good

**Pull request merged**